common/views/components/ArchiveBreadcrumb/ArchiveBreadcrumb.js

```diff
@@ -0,0 +1,302 @@
+// @flow
+import { useState, useEffect } from 'react';
+import { workLink } from '../../../services/catalogue/routes';
+import NextLink from 'next/link';
+import { classNames } from '../../../utils/classnames';
+import fetch from 'isomorphic-unfetch';
+import styled from 'styled-components';
+import DropdownButton from '../DropdownButton/DropdownButton';
+import Icon from '../Icon/Icon';
+
+const ArchiveBreadcrumbNav = styled.nav`
+  * {
+    font-family: 'helvetica neue' !important;
+    font-weight: normal !important;
+    font-size: 14px !important;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0 0 10px;
+    padding: 0;
+    display: inline-flex;
+    align-items: center;
+  }
+
+  li {
+    a {
+      text-decoration: none;
+      display: flex;
+      align-items: baseline;
+    }
+
+    &.is-last a {
+      background: #333;
+      color: white;
+      padding: 3px 8px;
+      border-radius: 3px;
+    }
+
+    .icon {
+      margin-right: 5px;
+      position: relative;
+      top: 2px;
+    }
+
+    position: relative;
+    cursor: pointer;
+    margin-right: 20px;
+
+    &:after {
+      position: absolute;
+      right: -15px;
+      top: 50%;
+      transform: translateY(-50%);
+      font-family: monospace;
+      color: #ccc;
+      content: '>';
+    }
+
+    &:last-child {
+      &:after {
+        display: none;
+      }
+    }
+
+    ul {
+      display: block;
+
+      li {
+        margin-bottom: 1px;
+
+        a {
+          padding: 3px 8px;
+          border-radius: 3px;
+
+          &:hover {
+            background: #333;
+            color: white;
+          }
+        }
+
+        &.is-active a {
+          background: #333;
+          color: white;
+        }
+
+        &:after {
+          display: none;
+        }
+
+        &:last-child {
+          &:after {
+            display: none;
+          }
+        }
+      }
+    }
+  }
+`;
+
+type Props = {|
+  work: { id: string },
```

Comment:
[Same as here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5301/files#r431665830), could we start using the work type?